src/WordPress.php

```diff
@@ -978,6 +978,10 @@ private function wp_download_and_config( $assoc_args ) {
 		// Added wp-config.php debug constants referred from https://codex.wordpress.org/Debugging_in_WordPress.
 		$extra_php = "if ( isset( \$_SERVER[\"HTTP_X_FORWARDED_PROTO\"] ) && \$_SERVER[\"HTTP_X_FORWARDED_PROTO\"] == \"https\" ) {\n	\$_SERVER[\"HTTPS\"] = \"on\";\n}\n\n// Enable WP_DEBUG mode.\ndefine( \"WP_DEBUG\", false );\n\n// Enable Debug logging to the /wp-content/debug.log file\ndefine( \"WP_DEBUG_LOG\", false );\n\n// Disable display of errors and warnings.\ndefine( \"WP_DEBUG_DISPLAY\", false );\n@ini_set( \"display_errors\", 0 );\n\n// Use dev versions of core JS and CSS files (only needed if you are modifying these core files)\ndefine( \"SCRIPT_DEBUG\", false );";
 
+		if ( 'subdom' === $this->site_data['app_sub_type'] || 'subdir' === $this->site_data['app_sub_type'] ) {
```

Comment:
A simpler condition could be:
```suggestion
		if ( 'wp' !== $this->site_data['app_sub_type'] ) {
```